content/game_process.tex

```diff
@@ -105,28 +105,27 @@ \subsubsection{Invalid Goal}
     \item When a team scores on themselves and there are no opponent robots on the field that are active (a definition of \emph{active} is given in \cref{sec:fallenrobots}).
 \end{enumerate}
 
-In these cases a goal is not scored (that is, the goal is ruled invalid), and the game will proceed with a Goal Kick (\cf \cref{sec:free_kick}). The head referee should also advice why the goal is invalid, such as by calling ``Not indirect'', ``Played with hands'' or ``No own goal''.
+In these cases a goal is not scored (that is, the goal is ruled invalid), and the game will proceed with a Goal Kick (\cf \cref{sec:free_kick}). The head referee should also advise why the goal is invalid, such as by calling ``Not indirect'', ``Played with hands'' or ``No own goal''.
 
 \subsubsection{Indirect Kick}
 \label{sec:indirect_kick}
 
-From any restart in play (Kick-off, Kick-in, Free kick) except for Penalty kick, the attacking team may only score a goal via an indirect kick.
-%, providing they have at least 3 active robots on the field at the time the restart occurs (or is awarded in the case of a free kick).
-A robot may not score a goal from a direct kick, including via deflections.
-The ball must be deliberately played-at a second time (by either another robot, or the same robot) before a goal may be scored.
-A deliberate play at the ball includes successfully kicking the ball, dribbling the ball (and subsequently leaving possession of the ball), or the goalkeeper playing at the ball with its hands.
-If a robot plays the ball to itself, this means that the ball must leave a circular area of at least \qty{0.75}{\metre} the robot before the ball is played a second time and to be considered as an indirect
-kick.
+The attacking team may only score a goal via indirect kick from any restart into \textit{playing} from a Free kick except for a Penalty kick. This rule applies also for the defending team in case of unexecuted Kick-off and the ball is free. 
```

Comment:
Isn't it clearer to just state: 
**Any team** may only score a goal via indirect kick from any restart into \textit{playing} **(including Free kicks)** except for a Penalty kick. This rule applies also for the defending team in case of unexecuted Kick-off and the ball is free.

Reply:
This would be another case and not allowing the defending team to score a direct goal if a goal kick has not been executed by the attacking team. Or also at the Kick in.

Reply:
Are the indirect requirements for attacking and defensive team at kickoff dealt with somewhere else? If not it still needs to be called out explicitly.
Are deflections considered indirect or not? If not, this should still be stated explicitly.

Reply:
Ok, I thought the idea was to only allow indirect kicks. So no goal direct from the goal kick or some kick ins

Reply:
There is a list which includes all the points directly below lines: 115 to 120